src/main/java/io/aiven/kafka/auth/audit/AuditorConfig.java

```diff
@@ -24,6 +24,10 @@
 public class AuditorConfig extends AbstractConfig {
 
     static final String AGGREGATION_PERIOD_CONF = "aiven.acl.authorizer.auditor.aggregation.period";
+    static final String AGGREGATION_GROUPING_CONF = "aiven.acl.authorizer.auditor.aggregation.grouping";
+
+    static final String AGGREGATION_GROUPING_PRINCIPAL = "Principal";
```

Comment:
Lets extract `AGGREGATION_GROUPING_PRINCIPAL` and `AGGREGATION_GROUPING_PRINCIPAL_AND_SOURCE_IP` in enum

Reply:
Makes sense. I'll do that.